lib/ansible/module_utils/basic.py

```diff
@@ -888,7 +888,7 @@ def is_special_selinux_path(self, path):
             f.close()
         except Exception:
             return (False, None)
-        path_mount_point = self.find_mount_point(path)
+        path_mount_point = to_text(self.find_mount_point(path), errors='surrogate_or_strict')
```

Comment:
I'll need to look over this in more depth, but we typically don't use text on the module side, either native or bytes. Whereas controller code is text or bytes.

We only use text on the module side if the API requires it. So I wonder if the change should be reversed. Do all comparisons in bytes.

Reply:
Ah, I didn't know that convention and which one I converted was arbitrary. Easy change.

Reply:
As I found out, `find_mount_point()` varies its return type with its input type (if given a bytestring, it returns a bytestring) and this function (`is_special_selinux_path()`) is sometimes called with a bytestring and sometimes with a string.

So we still need to force the output of `find_mount_point()` to be one or the other, for a consistent check. I've updated it to convert both sites of the equality check to be bytestrings.